scorecard/profile_data.py

```diff
@@ -1046,10 +1048,10 @@ def get_muni_specifics(cls, api_data):
         # If latest is missing, there are none to show.
         if latest_quarter is not None:
             keys = []
-            for q in xrange(latest_quarter['quarter'], 0, -1):
-                keys.append((latest_quarter['year'], q))
-            for q in xrange(4, 0, -1):
-                keys.append((latest_quarter['year']-1, q))
+            for q in range(int(latest_quarter['quarter']), 0, -1):
```

Comment:
I think we should rather fix quarter_tuple(...) to return ints as one would expect, rather than turn all of the floats it's generating into ints individually